### DISCUSSION TITLE
Stage Manager interferes with left positioning

### DISCUSSION THREAD

**Discussion by EugenioGrant** [rxhanson/RectanglePro-Community#293] [Ideas]
If I activate Stage Manager then I'm not able to snap a window on the left side as there is some space that does not allow Rectangle Pro to snap completely to the left... 

Has anyone seen this behaviour.

**Comment by rxhanson** [comment]
If I understand correctly, you don't want Rectangle Pro to leave space for the Stage Manager's recent apps area, and you can do this by setting the slider value to zero in the settings tab of the prefs window.

<img width="811" alt="Screenshot 2023-06-16 at 7 39 33 AM" src="https://github.com/rxhanson/RectanglePro-Community/assets/13651296/7918553e-429e-4463-9dc6-e2b613485fcd">

Let me know if that's not what you were looking for.

**Comment by markjirousek** [comment]
Seems like this option doesn't work anymore, unless I am missing something. But I'm pretty sure it worked before when I was using the non-pro version.

I mean pushing the slider to the far right to give room for the recent apps. Doesn't do anything.
